<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GpsSourceDialog</class>
 <widget class="QDialog" name="GpsSourceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>484</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Source Type</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cboSourceType">
       <item>
        <property name="text">
         <string>NMEA Serial</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>GPSd</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Serial Port</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Baud Rate</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cboSerialSpeed">
        <property name="currentIndex">
         <number>7</number>
        </property>
        <item>
         <property name="text">
          <string>1200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4800</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>9600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>19200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>38400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>57600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>115200</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Data Bits</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="cboSerialDataBits">
        <item>
         <property name="text">
          <string>8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Parity</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="cboSerialParity">
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Even</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Odd</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Stop Bits</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="cboSerialStopBits">
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1.5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Flow Control</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="cboSerialFlowControl">
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hardware</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Software XON/XOFF</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QPushButton" name="btnSerialTestConnection">
        <property name="text">
         <string>Test Connection</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Port</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cboSerialPort"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Mapping</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Marker Label</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lneMarkerLabel">
        <property name="text">
         <string>GPS0</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QCheckBox" name="chkKeepInFrame">
        <property name="text">
         <string>Keep location in frame</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="chkCenterMap">
        <property name="text">
         <string>Center map on location</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QCheckBox" name="chkMarkerVisible">
        <property name="text">
         <string>Marker visible</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>12</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GpsSourceDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GpsSourceDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
